test/org/zaproxy/zap/utils/LocaleUtilsUnitTest.java

```diff
@@ -160,4 +162,30 @@ public void shouldNotMatchInvalidMessagesPropertiesFilesWithCreateMessagesProper
             assertThat(file, pattern.matcher(file).matches(), is(equalTo(false)));
         }
     }
+
+    @Test
+    public void shoudAvailableLocalesBeNonEmpty() {
+        // Given
+        List<String> locales = LocaleUtils.getAvailableLocales();
+
+        // When
+        int localesSize = locales.size();
+
+        // Then
+        assertTrue(localesSize > 0);
+    }
+
+    @Test
+    public void shoudTheDefaultLocaleBeEnglish() {
```

Comment:
Probably better to call it "shouldHaveEnglishAsFirstAvailableLocale" instead. What matters most is that it's the first locale, not if it's default or not (it's not).